frontend/src/models/behaviors.js

```diff
@@ -26,3 +27,111 @@ export class RandomCardBehavior {
     return null;
   }
 }
+
+export class BasicRuleBasedBehaviour {
+  announcementToMake(possibleAnnouncements) {
+    // ToDo announce by "goodies"
+    /**
+     * for instance
+     * only one fehlsuit
+     * high number of trumps
+     * high trumps, such as 2x ten of hearts
+     * blank ace and starting
+     */
+    return null;
+  }
+
+  cardToPlay(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (!baseCard) {
+      /** It's our turn. Decide how to deal with cards */
+      return this.startingRule(hand, memory);
+    }
+    if (!baseCard.isTrump()) {
+      /** We need to serve a fehl card */
```

Comment:
Good point, but from a single player point it would be no good move.

I'm doing this on purpose right now because of the lack of known affinites.

This is a behaviour I might change when affinities are implemented. Obviously there are moves like "abwerfen" and "schmieren" but this is something I only do if I have much more info on the game, especially if I know my teammate. Or plan on winning a different trick with the "abwerfen" move.